composer/datasets/in_context_learning_evaluation.py

```diff
@@ -1234,11 +1235,13 @@ class InContextLearningCodeEvalDataset(InContextLearningDataset):
     def __init__(
         self,
         generations_per_sample: int,
-        pass_at_k: int = 1,
+        pass_at_k: Union[int, list[int]] = 1,
```

Comment:
why is pass at k a list now?

Reply:
ah once i got to the end i now understand. it is to compute multiple pass at ks without having to run this multiple times. 👍